Dockerfile.centos

```diff
@@ -74,6 +65,3 @@ RUN pip install -U cython
 
 # Install package
 RUN pip install /opt/file-metadata --upgrade
-
-# Test file-metadata import
-RUN python -c'import file_metadata; print(file_metadata.__version__)'
```

Comment:
We need some command to run in the end to test the successful install. At least an import test.